docs/contributors/adding-language-support/starter-code.mdx

```diff
@@ -1,30 +1,63 @@
 ---
 title: "Starter Code"
-description: "Learn about how to create starter code for a language"
+description: "Learn about how to implement starter code for a language"
 ---
 
 Starter code is the code that we provide to users when they start a challenge. It's the code that they'll be building on top of.
 
 [Here's an example](https://github.com/codecrafters-io/build-your-own-redis/tree/main/compiled_starters/python).
 
-## Required Files
-
-At the minimum, starter code is expected to have the following files:
-
-- `README.md`
-  - A README file that explains how to run the starter code & submit code to CodeCrafters.
-- `codecrafters.yml`
-  - A YAML file with the keys `language_pack` and `debug`.
-  - The name of this files depends on the challenge. `your_docker.sh` for Docker, `your_git.sh` for Git etc.
-- An executable script that runs the starter code.
-- Skeleton code that the user will build upon.
-  - This code must also include a section that can be uncommented to pass the first stage.
-- Any configuration files commonly required by the language / package managers.
-  - For example, `requirements.txt` for Python, `package.json` for JavaScript etc.
-- `.gitignore`
-  - A `.gitignore` file that ignores files commonly ignored by the language.
-- `.gitattributes`
-  - A `.gitattributes` file that sets the `text=auto` for all files (to avoid issues with line endings on Windows).
+## 1. Copy from Language Template
+
+Use [course-sdk](https://github.com/codecrafters-io/course-sdk) to copy the starter code skeleton and Dockerfile from the [language-templates](https://github.com/codecrafters-io/language-templates) repository.
+
+<Steps>
+<Step title="Clone build-your-own-<challenge>">
+  For example, if you’re adding support for the Kafka challenge, clone the [build-your-own-kafka](https://github.com/codecrafters-io/build-your-own-kafka) repository.
+
+</Step>
+
+<Step title="Run course-sdk add-language <language>">
+a. Verify that the language you want to add is available in the [languages](https://github.com/codecrafters-io/language-templates/tree/main/languages) folder in the language-templates repository.
```

Comment:
Let's remove this - `course-sdk` will automatically print an error message if the language isn't supported